db/models/topics.models.js

```diff
@@ -0,0 +1,7 @@
+const db = require("../connection");
+
+exports.requestAllTopics = () => {
```

Comment:
The name is ok here, but it might be confused with 'req' from the controller, "fetch" or "select" might be a better name here.